certified-connectors/Spinpanel/apiDefinition.swagger.json

```diff
@@ -0,0 +1,2047 @@
+                  "properties": {
+                    "objectType": {
+                      "type": "string",
+                      "description": "Object type of the subscription.",
+                      "title": "Object type"
+                    }
+                  },
+                  "description": "attributes"
+                }
+              }
+            }
+          }
+        },
+        "summary": "Get subscriptions",
+        "description": "Retrieves all subscriptions for the customer of a partner by their active directory tenant identifier.",
+        "parameters": [
+          {
+            "name": "partnerId",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "Spinpanel partner identifier.",
+            "x-ms-summary": "Spinpanel partner identifier.",
+            "x-ms-url-encoding" : "single"
+          },
+          {
+            "name": "customerId",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "Spinpanel customer identifier.",
+            "x-ms-summary": "Spinpanel customer identifier.",
+            "x-ms-url-encoding" : "single"
+          },
+          {
+            "name": "tenantId",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "Microsoft tenant Id of the customer.",
+            "x-ms-summary": "Microsoft tenant Id of the customer.",
+            "x-ms-url-encoding" : "single"
+          }
+        ],
+        "operationId": "GetSubscriptions"
+      },
+      "patch": {
+        "responses": {
+          "200": {
+            "description": "Ok",
+            "schema": {}
+          }
+        },
+        "summary": "Update subscription quantity",
+        "parameters": [
+          {
+            "name": "partnerId",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "Spinpanel partner id.",
+            "x-ms-summary": "Spinpanel partner id.",
+            "x-ms-url-encoding" : "single"
+          },
+          {
+            "name": "customerId",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "Spinpanel customer id.",
+            "x-ms-summary": "Spinpanel customer id.",
+            "x-ms-url-encoding" : "single"
+          },
+          {
+            "name": "tenantId",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "Microsoft tenant id of the customer.",
+            "x-ms-summary": "Microsoft tenant id of the customer.",
+            "x-ms-url-encoding" : "single"
+          },
+          {
+            "name": "quantityOnly",
+            "in": "query",
+            "required": true,
+            "type": "boolean",
+            "default": true,
+            "description": "",
```

Comment:
@VincentMejan Empty property values can be removed until unless you have a specific reason to keep them.

Reply:
Removed empty value